src/constants/constants.ts

```diff
@@ -0,0 +1,3 @@
+// action constants
+export const SET_KEYCLOAK_ROLES = 'setKeyCloakRoles'
```

Comment:
As these are used for store purposes only, can they be moved to the store folder?

Reply:
Yea, that makes sense. Will make a store constants for dispatch references!